### PR TITLE
Actual New Content: Shit Soup, Alchemy Jug, Gnomes, Redcaps, PP Milking, Tear Juice, Greater Aerosolize, etc etc.

### PR DESCRIPTION
**Pull request by josephpatrickgreen**
## About The Pull Request
**Gnome**
intrinsically magical race, t1 spells with 2 spell points (if arcyne potential upgrades to t2 (must not be h armor, m armor or dodge expert role), -1 constitution, -1 willpower, +1 luck, shunned race. debuffed heavily with -1 will and con

Gnomes were first shaped by Xylix, the Trickster, from the echoes of elven laughter and pure joy — small folk meant to amuse and inspire the greater folk. But the spark that gave them wit burned too fiercely; they grew restless under their creator's light, delighting in mischief that soured into malice. When their pranks turned cruel and their illusions began to twist the minds of mortals, polite society cast them out, banishing them to the shallow warrens between root and stone. There, in the dim humus of forgotten groves, they festered — no longer creators of joy, but of mockery. They learned to weave jests as deftly as blades, to turn truth into poison and laughter into a weapon. In their exile they came to worship their own cleverness, believing deceit itself to be the highest form of art. Xylix mourned their fall, yet would not unmake them, for even ruin has its place in the great design. Among the mortal folk, gnomes now skulk as tricksters, charmers, and whispering tempters — never fully trusted, yet never fully shunned, for their cunning remains a potent, perilous gift.
	
**Redcap**
despised race, +1 luck, +1 strength, -1 willpower, hat dipper trait. evil gnomes. They can dip any hat (not helms) in a blood puddle to give them a minor healing buff and temporary haste.

The Redcaps are said to be the bitter children of Xylix's earliest works — gnomes who once danced in the hollow roots of the world, singing hymns of cleverness and craft. Graggar the Rot-Father crept into their hidden warrens, whispering promises of iron, blood, and dominion. When Xylix abandoned them for turning merrymaking into cruelty He taught them that blood was the truest magic, and that only by tempering their hats in the viscera of the living could they remain free in a dying world. Thus were born the first Redcaps — gnomes who fed upon the pulse of others to keep their own hearts beating. Their laughter curdled into shrieks, their whimsy twisted into weapon, and their craft became slaughter. 	The cap, once a mark of gnomish ingenuity, became a vessel for vitae; it is said that if it ever dries, the Redcap withers and dies. 	Though they have long since severed their bond to Xylix, a faint echo of their maker lingers in their cruel wit and dark inventiveness.

This content is dual licensed specifically to ratwood 2.0

Other additions:

- adds genital milking for male and female into container sexcon
- readds cum reagent
- adds fem squirt reagent
- readds cum into container sexcon.
- readds shit soup (boil shit in water)
- fixes organ grinding results so that they are accessible. there was a function where eating an organ turned it into food but then back into non-food after making their grind results never accessible.
- makes acid collectable
- readds tearjuice to onions grind results
- adds a T4 spell greater aerosolize which just makes it smokebomb radius of reagent. good for mass health pot.
- replaces court magician poison bottle with "alchemy jug", this is a magic jug that fills itself with a chosen reagent to itself in small quantities per minute from a list of (fresh water 45u pm, salt water 45u pm, beer 15u pm, wine 7.5u pm, cider 7.5u pm, honey 3.25u pm, lemonade 15u pm, mayonnaise 7.5u pm, poison berry stew 3.25u pm, acid 1u pm)
- adds a new spell which fills randomized, gross smelling reagents to a container
main ingredient (60% of container)
secondary ingredient (30% of container)
tertiary ingredient (10% of container)

ingredients:
- yucky soup
- fish soup
- cabbage soup
- sewage
- shit soup

## Testing Evidence
it works in the character creator. then crashes (as is typical behavior when running through vs code these days with no file changes) so it works as best as it can

## Why It's Good For The Game
its actual content instead of ports and fixes, which means it can be expanded upon.

i dunno, I've been faking being a gnome by being verminfolk, maybe others wanna be gnomes. I'm a bit of a traditionalist when it comes to what I add, i dont reinvent the wheel, I just use the tried and true stuff from the book (dragon ttrpg) gnomes have intrinsic magic and are lucky, hence the fortune and few spell points. redcaps are like blood magic people

- genital milking, highly requested sexcon that fills container, why not. same with masturbating into container (less requested.
- shit soup is a non-poisonous yet still gross jest (for jesters and nuissances)
- tearjuice for onions is fun and has a cool effect dip your dagger in it!
- another t4 spell for wizards
- alchemy jug is a cool dnd item
- noxious brew spell is a stylized dnd spell of tashas caustic brew. Sewage is poisonous, it's better to add randomized yucky reagents for funny jests that are potentially harmless instead of poisoning people consistently with sewage.

### CODE REVIEW

**Comment by Alanii** [comment]
This PR does too much. It should be seperate PRs. Adding two races, fetish content, and some reagent changes into one PR invites far too many problems to solve than fix. 

If there is instability due to this PR, it will result in the entirety of the PR content being removed due to this. You have a higher liklihood of success by atomizing it.

Also I am very against the weird "shit soup" meme. Next you're going to tell me the PR adds "shit emote."

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * license. Explicit written permission from the copyright holder is
+ * required for any use, modification, or redistribution outside of
+ * the Ratwood-Vale/Ratwood-2.0 project.
+ *
+ * ------------------------------------------------------------------
+ * WHY THIS IS AGPLv3-COMPLIANT AND NOT A "RESTRICTION"
+ * ------------------------------------------------------------------
+ *
+ * This is a standard and fully compliant dual-licensing model used by
+ * numerous open source projects (e.g. MySQL, Qt, etc.).
```
Comment (noelle-lavenza) [inline code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm]: incorrect. you cannot stipulate "outside of [X]" in your dual licensing scheme. if it is AGPLv3 code then it is AGPLv3 code, that's the nature of copyleft. you are permitted to dual license it, but all that would do is allow a closed source codebase with no AGPLv3 code to use your code.

File: modular/Neu_Food/code/drinks/stew.dm [hunk 2]
```diff
@@ -168,3 +179,11 @@
 			M.add_nausea(3) // so one berry or one dose (one clunk of extracted poison, 5u) will make you really sick and a hair away from crit.
 			M.adjustToxLoss(2)
 	return ..()
+
+//Like murky water but two times as strong, stew metabolizes much faster so it is not deadly
+/datum/reagent/consumable/soup/poo/on_mob_life(mob/living/carbon/M)
+	if(HAS_TRAIT(M, TRAIT_NASTY_EATER)) // lets orcs and goblins drink bogwater
```
Comment (noelle-lavenza) [inline modular/Neu_Food/code/drinks/stew.dm]: it isn't bogwater it's shit soup, this comment is inaccurate

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+	if(!bloodpuddle)
+		return
+
+	if(sewrepair) //check if headwear is a hat
+		var/mob/living/carbon/human/H = user
+		H.visible_message(
+			span_notice("[H] begins to wipe up \the [bloodpuddle.name] with \the [src]."),
+			span_notice("I wipe up \the [bloodpuddle.name] with my [src]...")
+		)
+		if(do_after(H, 20, target = bloodpuddle))
```
Comment (noelle-lavenza) [inline code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm]:
```suggestion
		if(do_after(H, 2 SECONDS, target = bloodpuddle))
```

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+	// Search the turf for any blood puddle objects
+	var/obj/effect/decal/cleanable/blood/puddle/bloodpuddle = locate() in bloodspot
+	if(!bloodpuddle)
+		return
+
+	if(sewrepair) //check if headwear is a hat
+		var/mob/living/carbon/human/H = user
+		H.visible_message(
+			span_notice("[H] begins to wipe up \the [bloodpuddle.name] with \the [src]."),
+			span_notice("I wipe up \the [bloodpuddle.name] with my [src]...")
```
Comment (noelle-lavenza) [inline code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm]:
```suggestion
			span_notice("I wipe up \the [bloodpuddle] with my [src]...")
```
`\the [foo.name]` is invalid. `[foo]` on its own is the same as `\the [foo]` (except for some edge cases with capitalization; at the start of sentences it's `\The [foo]` instead), and the way you avoid the auto-insertion of `\the` is by using `[foo.name]`.

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+
+	// Search the turf for any blood puddle objects
+	var/obj/effect/decal/cleanable/blood/puddle/bloodpuddle = locate() in bloodspot
+	if(!bloodpuddle)
+		return
+
+	if(sewrepair) //check if headwear is a hat
+		var/mob/living/carbon/human/H = user
+		H.visible_message(
+			span_notice("[H] begins to wipe up \the [bloodpuddle.name] with \the [src]."),
```
Comment (noelle-lavenza) [inline code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm]:
```suggestion
			span_notice("[H] begins to wipe up \the [bloodpuddle] with \the [src]."),
```
see below comment

File: code/__DEFINES/roguetown.dm [hunk 6]
```diff
@@ -228,13 +230,17 @@ Balloon Alert / Floating Text defines
...
 
 #define RACES_DESPISED \
 	/datum/species/goblinp,\
+	/datum/species/fey/redcap,\
 
 #define RACES_CONSTRUCT \
 	/datum/species/construct/metal,\
 	/datum/species/construct/porcelain,\
+	/datum/species/fey/gnome,\
+	/datum/species/fey/redcap,\
```
Comment (noelle-lavenza) [inline code/__DEFINES/roguetown.dm]: why are they considered constructs? aren't they fucked up fey elves? sure they were created by xylix but i don't think that makes them a construct per se

**Comment by josephpatrickgreen** [comment]
> I already reviewed your past PR, and you refused to make requested changes. Per Our contribution guidelines, "2 - Maintainer decisions on game direction are absolute. If something on a PR is not up to a Maintainers standards and the author is unwilling to comply the PR should be closed.".
> 
> Combining all of your changes into a single PR and then trying to bypass maintainers entirely by pestering Useroth in DMs is horrendous behavior. The only reason I haven't shut this down off the bat, Is that Useroth has explicitly told me he'd handle it, so I intend to leave it to him.

dw useroth said he wants to review it himself

File: code/__DEFINES/roguetown.dm (hunk 6 above)
Comment (josephpatrickgreen) [reply code/__DEFINES/roguetown.dm]: it was a requested change so they cant be duke

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+	H.mind?.special_items.Add(/obj/item/clothing/head/roguetown/feycap)
+
+/datum/species/fey/redcap/check_roundstart_eligible()
+	return TRUE
+
+/obj/item/clothing/head/roguetown/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	if(!HAS_TRAIT(user, TRAIT_FEY_HAT_DIPPER))
+		return
+	if(!proximity || !check_allowed_items(target, target_self = 1))
```
[the remark this answers is not in the document]
Comment (josephpatrickgreen) [reply code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm]: 1 IS TRUE

File: modular/Neu_Food/code/drinks/stew.dm (hunk 2 above)
Comment (josephpatrickgreen) [reply modular/Neu_Food/code/drinks/stew.dm]: from port from original ratwood

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/gnome.dm [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+	They dwell in tangled roots, damp burrows, and half-collapsed ruins where they whisper curses. \
+	Gnomes take pride in sowing chaos among mortals—turning neighbor against neighbor, warping dreams, \
+	and preying upon the weak-willed. \
+	Their laughter is said to drive men mad, and their bargains always end in loss. \
+	Though physically unimposing, they possess a cunning beyond reckoning and wield magic and deceit as naturally as breathing. \
+	Among the the sentient, they are tolerated for their utility, but never trusted; \
+	to strike a deal with a gnome is to court ones own undoing. \
+	(+1 Fortune, -1 Constitution, -1 Willpower, Arcane T1 Trait (T2 if Arcyne Potential))"
+	expanded_desc = "Gnomes were first shaped by Xylix, the Trickster, from the echoes of elven laughter and pure joy — small folk meant to amuse and inspire the greater folk. \
+	But the spark that gave them wit burned too fiercely; they grew restless under their creators light, delighting in mischief that soured into malice. \
```
[the remark this answers is not in the document]
Comment (josephpatrickgreen) [reply code/modules/mob/living/carbon/human/species_types/roguetown/fey/gnome.dm]: cant put ' in desc

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+ * Copyright (c) 2025 Jerry Goldman
+ * Discord: "___.__.__.__.__.___"
+ *
+ * DUAL LICENSE NOTICE (AGPLv3-compliant):
+ *
+ * This file is made available under TWO possible licenses. You must
+ * choose ONE of them. You do NOT receive both simultaneously.
+ *
+ * ------------------------------------------------------------------
+ * 1) GNU AGPLv3 LICENSE (applies ONLY within Ratwood-Vale/Ratwood-2.0)
```
[the remark this answers is not in the document]
Comment (josephpatrickgreen) [reply code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm]: that's for one of the two licenses.

**Comment by josephpatrickgreen** [comment]
> This PR does too much. It should be seperate PRs. Adding two races, fetish content, and some reagent changes into one PR invites far too many problems to solve than fix.
> 
> If there is instability due to this PR, it will result in the entirety of the PR content being removed due to this. You have a higher liklihood of success by atomizing it.
> 
> Also I am very against the weird "shit soup" meme. Next you're going to tell me the PR adds "shit emote."

port from original ratwood

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/gnome.dm (hunk 8 above)
[the remark this answers is not in the document]
Comment (noelle-lavenza) [reply code/modules/mob/living/carbon/human/species_types/roguetown/fey/gnome.dm]:
```suggestion
	But the spark that gave them wit burned too fiercely; they grew restless under their creator&apos;s light, delighting in mischief that soured into malice. \
```
use the html entity, that should render fine

File: code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm (hunk 9 above)
[the remark this answers is not in the document]
Comment (Cyprex) [reply code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm]: Do not resolve conversations like this to hide issues. Your license is bogus, and you cannot restrictively license code like this

File: modular/Neu_Food/code/drinks/stew.dm (hunk 2 above)
Comment (noelle-lavenza) [reply modular/Neu_Food/code/drinks/stew.dm]: the comment says "lets orcs and goblins drink bogwater" because it is copy-and-pasted from the murky water code.

**Comment by kittysmooch** [comment]
> This pr contains language that states a segment of its code is only "valid agpl" when used within explicitly this org's repo, implying a restriction placed on the rights of your users.
> 
> This is either fully useless under section 7 or a violation of our license under section 10. https://www.gnu.org/licenses/agpl-3.0.en.html AGPL is infectious, the way to not have it apply to your code is to not work off AGPL code in the first place. You can sublicense code ONLY you wrote to other, less restrictive licenses, but you do not get to lock it up in a cage.
> 
> I understand the urge to limit duplication with the idea of avoiding "stealing" or "copies" from flooding byond, but, to be frank. That is what you are already doing, with my work and the work of hundreds of other people before you. Trying to section off only YOUR stuff is both a violation of the license we both work under, and an insult to the people whose work you are using for something they may or may not condone.
> 
> The reason this game is still alive in the first place, the reason ratwood exists as a concept, is because of code shared with the single limitation that you cannot just "take your stuff and go private". That is why we can survive servers dying, hosts melting down, or codebases exploding. You take my work, use it for cool and epic furry porn, and you don't even have the decency to keep the god damn game alive?

thanks for stepping up. you and the other tg maints should be aware if you aren't already that this repo is probably already in active violation of tgs licenses via another merged pr that packed code into the GitHub secrets folder

**Comment by Bomberman66Dev** [comment]
<unatomized
<untested beyond compilation
<nontent
<pr body written by jordan chatgpeterson
<stonewalling maintainers
<bothering the host like a selfish little fuck
<a single other merged pr by the author was merged
<literally illegal licensing
130528 - SoyBooru
OP you should keys immediately

**Comment by Bomberman66Dev** [comment]
<hello i am the real author of this PR
137991 - SoyBooru(1)

**Comment by Bomberman66Dev** [comment]
Oh… buckle up. Here’s a very harsh review of this PR, point by point. I’ve organized it in categories for clarity, because this PR is a dumpster fire in nearly every dimension:


---

1. Licensing / Legal Issues

The PR explicitly tries to dual-license AGPL content as “ratwood 2.0”. This is illegal and unmergeable. You cannot relicense AGPL content like this without violating copyright.

The author stonewalled maintainers, which is a huge red flag. This is not a negotiable matter; it alone is sufficient reason to reject the PR outright.


Verdict: 🚫 Immediate rejection. No discussion needed.


---

2. Scope & Atomicity

This PR is completely unatomized:

Adds new races (gnome, redcap)

Adds spells (T4 aerosolize, noxious brew)

Adds “container sexcon” features, genital milking, cum reagents, fem squirt reagents

Readds shit soup, tearjuice, organ grind fixes

Adds acid collection

Replaces items (alchemy jug)

Changes spell mechanics



This is at least 5+ completely unrelated PRs squashed into one. This makes code review impossible, introduces enormous risk, and violates standard PR hygiene.

Verdict: 🚫 Needs to be split into atomic PRs by feature, e.g., one for races, one for spells, one for reagents/sexcon features, one for items, etc.


---

3. Code Quality & Maintainability

No code snippets or diffs are included in the description—just prose. Impossible to review actual implementation.

Feature descriptions are informal, whimsical, and not developer-friendly:

> “i dunno, I've been faking being a gnome by being verminfolk”
“highly requested sexcon that fills container, why not”



This is not professional for a codebase, especially an AGPL project.

No mention of tests, error handling, edge cases beyond a vague VS Code crash.

Many features seem game-breaking or trollish (shit soup, sewage, genital milking), which may be intended humor, but could break the core balance of the game.


Verdict: 🚫 Code quality is nonexistent, documentation is insufficient, and features are unsafe/fragile.


---

**
122714 - SoyBooru

**Comment by RadiantFlash7** [comment]
@Bomberman66Dev I'm going to have to remind you of our contributor guidelines: 4 - Try to be civil while discussing PRs. We are all here working for the same goal and because we love what we do, remember that!
Yes, This PR has problems. MANY problems. I have a massive disagreeance with the author due to their behavior and the way this is currently being handled. But you still need to act civilly, which means. less meme posting and insults.

Rebuke for your less then civil behavior aside- Your harsh review does hit the mark spot on.

**Comment by kepesk1** [comment]
> > This pr contains language that states a segment of its code is only "valid agpl" when used within explicitly this org's repo, implying a restriction placed on the rights of your users.
> > This is either fully useless under section 7 or a violation of our license under section 10. https://www.gnu.org/licenses/agpl-3.0.en.html AGPL is infectious, the way to not have it apply to your code is to not work off AGPL code in the first place. You can sublicense code ONLY you wrote to other, less restrictive licenses, but you do not get to lock it up in a cage.
> > I understand the urge to limit duplication with the idea of avoiding "stealing" or "copies" from flooding byond, but, to be frank. That is what you are already doing, with my work and the work of hundreds of other people before you. Trying to section off only YOUR stuff is both a violation of the license we both work under, and an insult to the people whose work you are using for something they may or may not condone.
> > The reason this game is still alive in the first place, the reason ratwood exists as a concept, is because of code shared with the single limitation that you cannot just "take your stuff and go private". That is why we can survive servers dying, hosts melting down, or codebases exploding. You take my work, use it for cool and epic furry porn, and you don't even have the decency to keep the god damn game alive?
> 
> thanks for stepping up. you and the other tg maints should be aware if you aren't already that this repo is probably already in active violation of tgs licenses via another merged pr that packed code into the GitHub secrets folder

"Probably already in active violation," that's great and all -- let us know when you find that violation then, since it appears you "probably" think it exists. Fortunately the whole repo is public, so you should have no problem finding it since it, apparently, "actively" exists? 👍

**Comment by LemonInTheDark** [comment]
> Hey Lemon

Hello gravy. I assumed as much, s why I'm mostly talking to him rather then like, calling "his manager" or whatever. The pr does smell kinda like a feature dump, which is not typically a good sign of well, good practice. Surprised your host would do that, but I have long since learned to not stick my nose in downstream drama/problems.

You don't need to apologize s fine, this stuff just gets me quite heated for reasons elaborated in my towerpost. I started commenting here because of a maintainer of yours talking about how it sucked so I didn't think you were all like gung ho for this bull.

**Comment by RadiantFlash7** [comment]
I said I'd let Useroth handle this, But frankly leaving this PR up to be considered is a bad look and reflects rather badly on Ratwood2. The lack of atomization of this PR, the flagrant breaches of contributor guidelines and outright ignoring change requests are bad enough that it would be closed normally. The Licensing issues that have been brushed off however, are massive sticking point and violation of our AGPL license. Due to this, The entire maintainer team has agreed this PR should be closed.

@Useroth If you feel it should be otherwise, talk to us in the maintainer channel on discord.

**Comment by josephpatrickgreen** [comment]
> This pr contains language that states a segment of its code is only "valid agpl" when used within explicitly this org's repo, implying a restriction placed on the rights of your users.
> 
> This is either fully useless under section 7 or a violation of our license under section 10. https://www.gnu.org/licenses/agpl-3.0.en.html AGPL is infectious, the way to not have it apply to your code is to not work off AGPL code in the first place. You can sublicense code ONLY you wrote to other, less restrictive licenses, but you do not get to lock it up in a cage.
> 
> I understand the urge to limit duplication with the idea of avoiding "stealing" or "copies" from flooding byond, but, to be frank. That is what you are already doing, with my work and the work of hundreds of other people before you. Trying to section off only YOUR stuff is both a violation of the license we both work under, and an insult to the people whose work you are using for something they may or may not condone.
> 
> The reason this game is still alive in the first place, the reason ratwood exists as a concept, is because of code shared with the single limitation that you cannot just "take your stuff and go private". That is why we can survive servers dying, hosts melting down, or codebases exploding. You take my work, use it for cool and epic furry porn, and you don't even have the decency to keep the god damn game alive?

Now that everything is closed, to comment on this, there are servers using my code commercially, without commenting on anything or anyone specifically yet, that's my reasoning

**Comment by GravyMaker** [comment]
In principle, open-source code _can_ be used commercially - I don't think the AGPL includes a noncommercial clause, and I know that the GPL doesn't. Presumably you mean it's being used in a proprietary, closed-source codebase, in which case they're already in violation of the licence, and trying to add a dual-licensing stipulation which isn't in concord with the AGPL wouldn't help matters anyway, even if it _were_ contractually valid.

**Comment by noelle-lavenza** [comment]
> > This pr contains language that states a segment of its code is only "valid agpl" when used within explicitly this org's repo, implying a restriction placed on the rights of your users.
> > This is either fully useless under section 7 or a violation of our license under section 10. https://www.gnu.org/licenses/agpl-3.0.en.html AGPL is infectious, the way to not have it apply to your code is to not work off AGPL code in the first place. You can sublicense code ONLY you wrote to other, less restrictive licenses, but you do not get to lock it up in a cage.
> > I understand the urge to limit duplication with the idea of avoiding "stealing" or "copies" from flooding byond, but, to be frank. That is what you are already doing, with my work and the work of hundreds of other people before you. Trying to section off only YOUR stuff is both a violation of the license we both work under, and an insult to the people whose work you are using for something they may or may not condone.
> > The reason this game is still alive in the first place, the reason ratwood exists as a concept, is because of code shared with the single limitation that you cannot just "take your stuff and go private". That is why we can survive servers dying, hosts melting down, or codebases exploding. You take my work, use it for cool and epic furry porn, and you don't even have the decency to keep the god damn game alive?
> 
> Now that everything is closed, to comment on this, there are servers using my code commercially, without commenting on anything or anyone specifically yet, that's my reasoning

i don't think this is true, i think this is either a misunderstanding or a willful misinterpretation, and given the history of willful misinterpretations on the license issue so far i am not inclined to believe it was a misunderstanding.

**Comment by crucialAmnesiac** [comment]
copyright for fucking shit soup LMFAO
